drawers/SampleDrawer.js

```diff
@@ -1,6 +1,7 @@
-import { Typography } from "@material-ui/core";
+import { Typography } from "@mui/material";
 
 const SampleDrawer = () => (
+  // eslint-disable-next-line
```

Comment:
Why is linting unhappy?

Reply:
Doesn't like the apostrophe in " I'm " apparently.

Reply:
What? 🤣🤣 my gosh

Reply:
Ok, we should fix this with &apos; as is suggested: https://stackoverflow.com/a/45898131

There is a good reason for this detailed here: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md#prevent-invalid-characters-from-appearing-in-markup-reactno-unescaped-entities